src/lib/notifications/hooks/useNotifications.common.js

```diff
@@ -0,0 +1,87 @@
+import { useCallback, useEffect, useMemo, useState } from 'react'
+import usePermissions from '../../../components/permissions/hooks/usePermissions'
+import { Permissions } from '../../../components/permissions/types'
+import { NotificationsAPI } from '../api/NotificationsApi'
+import { useUserStorage } from '../../../lib/wallet/GoodWalletProvider'
+
+const getStoreProperty = (userStorage, property) => {
+  if (!userStorage) {
+    return null
+  }
+
+  return userStorage.userProperties.getLocal(property)
+}
+
+export const getCategory = notification => {
+  const { payload } = notification || {}
+  const { category } = payload || {}
+
+  return category
+}
+
+export const useStoreProperty = property => {
```

Comment:
we may have this hook already please check. if not place it the same file where we have useUserStorage

Reply:
placed